cli/sawtooth_cli/block.py

```diff
@@ -30,8 +29,7 @@ def add_block_parser(subparsers, parent_parser):
     grand_parsers = parser.add_subparsers(title='grandchildcommands',
                                           dest='subcommand')
     grand_parsers.required = True
-    epilog = '''
-    details:
+    epilog = '''details:
```

Comment:
The epilog output doesn't look right when output. I think we need to use another formatter (the raw one probably) so that epilog is output properly.  If we don't address it here, we should open up a JIRA task to go through the CLI and make it all consistent.